packages/instant/src/components/zero_ex_instant_provider.tsx

```diff
@@ -121,6 +123,20 @@ export class ZeroExInstantProvider extends React.Component<ZeroExInstantProvider
         gasPriceEstimator.getGasInfoAsync();
         // tslint:disable-next-line:no-floating-promises
         this._flashErrorIfWrongNetwork();
+
+        // Analytics
+        if (this.props.disableAnalyticsTracking) {
+            disableAnalytics();
```

Comment:
TODO: Need to have this get reset if they call a subsequent render to enable analytics